draft-ietf-quic-tls.md

```diff
@@ -552,7 +552,7 @@ older than 1.3 is negotiated.
 
 QUIC requires that the initial handshake packet from a client fit within the
 payload of a single packet.  The size limits on QUIC packets mean that a record
-containing a ClientHello needs to fit within 1171 octets.
+containing a ClientHello needs to fit within 1170 octets.
```

Comment:
An 1170 octet ClientHello will have a two octet length, so this needs to be 1169.

Reply:
At the moment the PR states that Payload Length field is a varint, and therefore the minimum overhead is 1 octet. Do you think that we should change it to a fix length of 2 octets?

Reply:
If the packet is this big, then the length will be 2 octets.  You can't have encode 1170 (or 1169) in one octet.  Obviously, if the ClientHello is smaller, then the length might only be 1 octet, but I think that one octet is impossible.  ClientHello.random is 32 octets and either a shares is at least 32 octets or a PSK binder is that big too.

Reply:
Ah! Thank you for very much for the clarification.